[rank: 0] Global seed set to 1
/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/lightning/fabric/plugins/environments/slurm.py:165: PossibleUserWarning: The `srun` command is available on your system but is not used. HINT: If your intention is to run Lightning on SLURM, prepend your python command with `srun` like so: srun python train.py config_file=conf/l8biome.yaml module.model= ...
  rank_zero_warn(
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/lightning/fabric/plugins/environments/slurm.py:165: PossibleUserWarning: The `srun` command is available on your system but is not used. HINT: If your intention is to run Lightning on SLURM, prepend your python command with `srun` like so: srun python train.py config_file=conf/l8biome.yaml module.model= ...
  rank_zero_warn(
You are using a CUDA device ('NVIDIA A100 80GB PCIe') that has Tensor Cores. To properly utilize them, you should set `torch.set_float32_matmul_precision('medium' | 'high')` which will trade-off precision for performance. For more details, read https://pytorch.org/docs/stable/generated/torch.set_float32_matmul_precision.html#torch.set_float32_matmul_precision
/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:612: UserWarning: Checkpoint directory /scratch.local/yichia3/output/l8biome/l8biome_unet_resnet50_0.001_ce_moco exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name          | Type             | Params
---------------------------------------------------
0 | model         | Unet             | 32.5 M
1 | loss          | CrossEntropyLoss | 0     
2 | train_metrics | MetricCollection | 0     
3 | val_metrics   | MetricCollection | 0     
4 | test_metrics  | MetricCollection | 0     
---------------------------------------------------
32.5 M    Trainable params
0         Non-trainable params
32.5 M    Total params
130.187   Total estimated model params size (MB)
WARNING! The experiment directory, /scratch.local/yichia3/output/l8biome/l8biome_unet_resnet50_0.001_ce_moco, already exists, we might overwrite data in it!
Sanity Checking: 0it [00:00, ?it/s]Sanity Checking:   0%|          | 0/2 [00:00<?, ?it/s]Sanity Checking DataLoader 0:   0%|          | 0/2 [00:00<?, ?it/s]/projects/dali/spack/var/spack/environments/dali/.spack-env/view/lib/python3.10/site-packages/matplotlib/cm.py:478: RuntimeWarning: invalid value encountered in cast
  xx = (xx * 255).astype(np.uint8)
Sanity Checking DataLoader 0:  50%|█████     | 1/2 [00:02<00:02,  2.01s/it]Sanity Checking DataLoader 0: 100%|██████████| 2/2 [00:02<00:00,  1.11s/it]                                                                           Training: 0it [00:00, ?it/s]Training:   0%|          | 0/781 [00:00<?, ?it/s]Epoch 0:   0%|          | 0/781 [00:00<?, ?it/s] Epoch 0:   0%|          | 1/781 [00:02<28:44,  2.21s/it]Epoch 0:   0%|          | 1/781 [00:02<28:44,  2.21s/it, v_num=2_3]Epoch 0:   0%|          | 2/781 [00:02<16:24,  1.26s/it, v_num=2_3]Epoch 0:   0%|          | 2/781 [00:02<16:24,  1.26s/it, v_num=2_3]Epoch 0:   0%|          | 3/781 [00:02<11:39,  1.11it/s, v_num=2_3]Epoch 0:   0%|          | 3/781 [00:02<11:39,  1.11it/s, v_num=2_3]Epoch 0:   1%|          | 4/781 [00:02<09:17,  1.39it/s, v_num=2_3]Epoch 0:   1%|          | 4/781 [00:02<09:17,  1.39it/s, v_num=2_3]Epoch 0:   1%|          | 5/781 [00:02<07:45,  1.67it/s, v_num=2_3]Epoch 0:   1%|          | 5/781 [00:02<07:45,  1.67it/s, v_num=2_3]Epoch 0:   1%|          | 6/781 [00:03<06:42,  1.93it/s, v_num=2_3]